specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/networkInterface.json

```diff
@@ -1160,6 +1160,11 @@
           "readOnly": true,
           "$ref": "./network.json#/definitions/ProvisioningState",
           "description": "The provisioning state of the network interface resource."
+        },
+        "workloadType": {
```

Comment:
Does the value of property is enumerable?

Reply:
It will be a string